src/content/app-platform/apps/_index.md

```diff
@@ -1,8 +1,11 @@
 ---
-linkTitle: Apps
-title: Apps
+linkTitle: Managed Apps
+title: Managed Apps
 description: Documentation for a selection of the most popular managed apps by Giant Swarm.
-weight: 50
+last_review_date: 2021-07-07
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-halo
```

Comment:
Owner is required. Not sure what the plan is for this page.

@giantswarm/team-halo Are you OK to take it?

Reply:
ye